### PR TITLE
Some small cleanups in limel-portal

### PR DESCRIPTION
**Pull request by adrianschmidt**
## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/portal/portal.tsx [hunk 1]
```diff
@@ -44,38 +44,38 @@ export class Portal {
      * Decides which direction the portal content should open.
      */
     @Prop({ reflect: true })
-    public openDirection: OpenDirection = 'bottom';
+    public openDirection?: OpenDirection = 'bottom';
```
Comment (Befkadu1) [inline src/components/portal/portal.tsx:47]: Does it really matter adding `?` here? It seems unnecessary.
`openDirection= 'bottom'` when no value is passed to this component anyway. The default values here can be overridden by specifying a value for a prop with a default value.
The same comment for the changes on 53, 65, 71, 78 and 87

**Comment by Befkadu1** [comment]
This is not a comment in this PR rather a question. You can omit it if you don't agree. I saw the other PR you made for `lime-tooltip` [here](https://github.com/Lundalogik/lime-elements/pull/2435/commits) and added `-basic` to filename of basic example. Maybe good to do it here as well?

File: src/components/portal/portal.tsx (hunk 1 above)
Comment (adrianschmidt) [reply src/components/portal/portal.tsx:47]: Yeah, reading the typescript docs [here](https://www.typescriptlang.org/docs/handbook/2/objects.html#optional-properties), it seems like `?` should be used when a property is optional but there is _not_ a default value. So I guess this is unnecessary, yes. I'll remove it again 👍

Comment (adrianschmidt) [reply src/components/portal/portal.tsx]: ⚡

I dropped this commit entirely 👍

File: src/components/portal/portal.tsx
[the remark this answers is not in the document]
Comment (adrianschmidt) [reply src/components/portal/portal.tsx:1]: ⚡

**Review by Befkadu1 — APPROVED** [review]
👏

**Pull request merged**